mimoto-default.properties

```diff
@@ -189,7 +189,7 @@ mosip.openid.issuers=openid-issuers-config.json
 # Configurations related to openid4vc 
 mosip.openid.issuers=openid-issuers-config.json
 GET_TOKEN=${public.internet.url}/v1/esignet/login/oauth/token
-mosip.oidc.esignet.token.endpoint=https://api-internal.dev1.mosip.net/v1/esignet/oauth/token
+mosip.oidc.esignet.token.endpoint=https://api-internal.dev1.mosip.net/v2/esignet/oauth/token
```

Comment:
Please use variables instead of hardcording domain

Reply:
testing is going on with this, we are in development stage @syedsalman3753